### DISCUSSION TITLE
Can't use recently transferred-in domain - "DNSSEC must be disabled

### DISCUSSION THREAD

**Discussion by Diehard0413** [vercel/community#6929] [Account Issue]
### Question

I am trying to use a domain I have recently purchased but Vercel gives me this error.

"Your domain's nameservers are pointing to Vercel, but you must disable DNSSEC with your domain registrar in order for your nameservers to resolve globally."

**Comment by AlbertMarashi** [comment]
Also having this issue after recently transferring in domain to vercel?

[a comment by an automated account is not shown]
